### ISSUE TITLE
Consistent parsing problem with latest stable phabricator

### ISSUE THREAD

**Issue by berkus** [etcinit/phabulous#21]
```
Jan 27 16:30:35 server1 phabulous[13013]: 2016/01/27 16:30:35 [Recovery] panic recovered:
Jan 27 16:30:35 server1 phabulous[13013]: POST /v1/feed/receive HTTP/1.1
Jan 27 16:30:35 server1 phabulous[13013]: Host: 127.0.0.1:8086
Jan 27 16:30:35 server1 phabulous[13013]: Accept: */*
Jan 27 16:28:33 server1 phabulous[13013]: Content-Type: application/x-www-form-urlencoded
Jan 27 16:28:33 server1 phabulous[13013]: json: cannot unmarshal array into Go value of type responses.PHIDQueryResponse
Jan 27 16:28:33 server1 phabulous[13013]: /usr/local/Cellar/go/1.5.1/libexec/src/runtime/panic.go:423 (0x42ab99)
Jan 27 16:28:33 server1 phabulous[13013]: /Users/berkus/Tools/go/src/github.com/etcinit/phabulous/app/controllers/feed.go:49 (0x546592)
Jan 27 16:28:33 server1 phabulous[13013]: /Users/berkus/Tools/go/src/github.com/etcinit/phabulous/app/controllers/feed.go:29 (0x547cea)
Jan 27 16:28:33 server1 phabulous[13013]: /Users/berkus/Tools/go/src/github.com/gin-gonic/gin/context.go:97 (0x5481ba)
Jan 27 16:28:33 server1 phabulous[13013]: /Users/berkus/Tools/go/src/github.com/gin-gonic/gin/recovery.go:45 (0x55a931)
Jan 27 16:28:33 server1 phabulous[13013]: /Users/berkus/Tools/go/src/github.com/gin-gonic/gin/context.go:97 (0x5481ba)
Jan 27 16:28:33 server1 phabulous[13013]: /Users/berkus/Tools/go/src/github.com/gin-gonic/gin/logger.go:66 (0x559bda)
Jan 27 16:28:33 server1 phabulous[13013]: /Users/berkus/Tools/go/src/github.com/gin-gonic/gin/context.go:97 (0x5481ba)
Jan 27 16:28:33 server1 phabulous[13013]: /Users/berkus/Tools/go/src/github.com/gin-gonic/gin/gin.go:294 (0x54eff2)
Jan 27 16:28:33 server1 phabulous[13013]: /Users/berkus/Tools/go/src/github.com/gin-gonic/gin/gin.go:275 (0x54ec27)
Jan 27 16:28:33 server1 phabulous[13013]: /usr/local/Cellar/go/1.5.1/libexec/src/net/http/server.go:1862 (0x64d6ae)
Jan 27 16:28:33 server1 phabulous[13013]: /usr/local/Cellar/go/1.5.1/libexec/src/net/http/server.go:1361 (0x64acee)
Jan 27 16:28:33 server1 phabulous[13013]: /usr/local/Cellar/go/1.5.1/libexec/src/runtime/asm_amd64.s:1696 (0x45c111)
Jan 27 16:28:33 server1 phabulous[13013]:
Jan 27 16:28:33 server1 phabulous[13013]: [GIN] 2016/01/27 - 16:28:33 | 500 |  206.227277ms | 127.0.0.1 |   POST    /v1/feed/receive
```

**Comment by berkus** [comment]
There isn't any more debug information as to what the payload was. I'll try to add that.

**Comment by etcinit** [comment]
Seems to be related or the same as #27

**Comment by schemar** [comment]
This should be fixed in the upcoming 3.0.0 release.

**Comment by berkus** [comment]
Thanks!
